etcdctl/ctlv3/command/make_mirror_command.go

```diff
@@ -71,8 +71,9 @@ func makeMirrorCommandFunc(cmd *cobra.Command, args []string) {
 		cacert:            mmcacert,
 		insecureTransport: mminsecureTr,
 	}
+	discovery := discoveryCfgFromCmd(cmd)
```

Comment:
the destination is already given on the command line, I don't think discovery makes sense here

Reply:
removed